src/templates/app/index.html

```diff
@@ -217,7 +217,7 @@
           <dl class="app-metadata-categories">
             <dt>{{ _('Categories') }}</dt>
             {% for category in CATEGORIES %}
-              {% if indexOf(this.categories, category.slug) != -1 %}
+              {% if this.categories and indexOf(this.categories, category.slug) != -1 %}
```

Comment:
Damnit you're right. Too busy with the expand toggle issue I should've seen its container.